docker/lr-transcript_utils/python/process_quant_files_into_tsv.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+
+# Read in data from individual quant files:
+import sys
+import glob
+import csv
+import time
+import os
+
+import anndata
+import scanpy as sc
+
+# Right now we accept only 1 argument - the directory in which the quant files live:
+if len(sys.argv) < 2:
```

Comment:
If a user specifies more than 1 file i think the script will ignore anything other than the first, which is fine but might be good to write it as 
```
# Right now we accept only 1 argument - the directory in which the quant files live:
    if len(sys.argv) != 2:
```
to alert the user that they shouldn't provide anything more than one file similar to docker/lr-transcript_utils/python/populate_zmw_sam_tag.py